contents/tutorials/when-and-how-to-run-group-targeted-a-b-tests.md

```diff
@@ -0,0 +1,88 @@
+---
+title: When and how to run group-targeted A/B tests
```

Comment:
This is a good title, but you should tighten up the slug.

- running-group-targeted-ab-tests
- group-targeted-ab-tests

The "how and when to run" part isn't super relevant for the the slug.